src/corona_hakab_model/consts.py

```diff
@@ -1,38 +1,119 @@
+    "symptomatic_infection_ratio": 0.75,
+    # probability of an infected asymptomatic agent infecting others
+    "asymptomatic_infection_ratio": 0.25,
+    # probability of an infected silent agent infecting others
+    "silent_infection_ratio": 0.3,  # todo i made this up, need to get the real number
+    # base r0 of the disease
+    "r0": 2.4,
+    # isolation policy
+    # todo why does this exist? doesn't the policy set this? at least make this an enum
+    # note not to set both home isolation and full isolation true
+    # whether to isolation detected agents to their homes (allow familial contact)
+    "home_isolation_sicks": False,
+    # whether to isolation detected agents fully (no contact)
+    "full_isolation_sicks": False,
+    # how many of the infected agents are actually caught and isolated
+    "caught_sicks_ratio": 0.3,
+    # policy stats
+    # todo this reeeeally shouldn't be hard-coded
+    # defines whether or not to apply a isolation (work shut-down)
+    "active_isolation": True,
+    # the date to stop work at
+    "stop_work_days": 40,
+    # the date to resume work at
+    "resume_work_days": 80,
+    # social stats
+    # the average family size
+    "family_size_distribution": rv_discrete(
+        1, 7, name="family", values=([1, 2, 3, 4, 5, 6, 7], [0.095, 0.227, 0.167, 0.184, 0.165, 0.081, 0.081])
+    ),  # the average workplace size
+    # work circles size distribution
+    "work_size_distribution": dist(30, 80),  # todo replace with distribution
+    # work scale factor (1/alpha)
+    "work_scale_factor": 40,
+    # the average amount of stranger contacts per person
+    "average_amount_of_strangers": 200,  # todo replace with distribution
+    # strangers scale factor (1/alpha)
+    "strangers_scale_factor": 150,
+    "school_scale_factor": 100,
+    # relative strengths of each connection (in terms of infection chance)
+    # todo so if all these strength are relative only to each other (and nothing else), whe are none of them 1?
+    "family_strength_not_workers": 0.75,
+    "family_strength": 1,
+    "work_strength": 0.1,
+    "stranger_strength": 0.01,
+    "school_strength": 0.1,
+    "detection_rate": 0.7,
+}
+
+ConstParameters = namedtuple("ConstParameters", sorted(default_parameters),
+                             defaults=[default_parameters[key] for key in sorted(default_parameters)])
+
+
+class Consts(ConstParameters):
+    __slots__ = ()
+    @staticmethod
+    def from_file(param_path):
+        """
+        Load parameters from file and return Consts object with those values.
+
+        No need to sanitize the eval'd data as we disabled __builtins__ and only passed specific functions
+        Documentation about what is allowed and not allowed can be found at the top of this page.
+        """
+        with open(param_path, "rt") as read_file:
+            data = read_file.read()
+
+        parameters = eval(data, {"__builtins__": None, "dist": dist, "rv_discrete": rv_discrete})
+
```

Comment:
If the researches send us some unhashable data types (lists or dicts), the program will crash in expected places, maybe we should check here that the return value is hashable?

Reply:
Will add back "sanitize_parameters" to check exactly that

Reply:
Done